### ISSUE TITLE
Unexpected error page after set marks

### ISSUE THREAD

**Issue by bazio101** [kgaut/mespronos#119]
Version instaled: 8.x-1.0-alpha14+71-dev (2018-Jun-16) with composer.
mespronos_group not installed

Created sport, league, teams, league_day, games and bets with success.
After I set marks, when I go to these pages:

1. https...mespronos/leagues/1
2. https...mespronos/ranking
3. https...mespronos/day/1

I always get the following error:

Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: You have requested a non-existent service "kgaut_tools.stringcleaner". in Drupal\Component\DependencyInjection\Container->get() (line 153 of /var/www/htdocs/mybetsite/web/core/lib/Drupal/Component/DependencyInjection/Container.php).

I had to disable "General Ranking Block" to reach the front page
Any ideas?

**Comment by bazio101** [comment]
I've just found kgaut_tools, your toolbox module for drupal 8, here on github.
I suppose this is the missing "non-existent service".
I'll try to install it and report back.

**Comment by bazio101** [comment]
No luck after kgaut_tools installation.
The only thing changed is the error message, which is much larger now:

``Error: Call to a member function transformDimensions() on null in template_preprocess_image_style() (line 305 of /var/www/htdocs/mybetsite/web/core/modules/image/image.module)  
#0 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Theme/ThemeManager.php(287): template_preprocess_image_style(Array, 'image_style', Array)
 #1 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(431): Drupal\Core\Theme\ThemeManager->render('image_style', Array) 
 #2 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(200): Drupal\Core\Render\Renderer->doRender(Array, false) 
#3 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Template/TwigExtension.php(501): Drupal\Core\Render\Renderer->render(Array) 
#4 /var/www/htdocs/mybetsite/web/sites/default/files/php/twig/5ef8e62239956_user-ranking.html.twig_1-jlrZfk8YOhdOH8DxyK2lVIt/FpRYjhe8FQFgalHI94nt0M7MVjjFYxihx3ShXFX8UTE.php(63): Drupal\Core\Template\TwigExtension->escapeFilter(Object(Drupal\Core\Template\TwigEnvironment), Array, 'html', NULL, true) 
#5 /var/www/htdocs/mybetsite/vendor/twig/twig/src/Template.php(455): __TwigTemplate_6f453e5e1a67a2b9103ec444658514abe95599bfc88a47ec702803fc4fe57bc1->doDisplay(Array, Array) 
#6 /var/www/htdocs/mybetsite/vendor/twig/twig/src/Template.php(422): Twig\Template->displayWithErrorHandling(Array, Array) 
#7 /var/www/htdocs/mybetsite/vendor/twig/twig/src/Template.php(434): Twig\Template->display(Array) 
#8 /var/www/htdocs/mybetsite/web/core/themes/engines/twig/twig.engine(64): Twig\Template->render(Array) 
#9 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Theme/ThemeManager.php(384): twig_render_template('modules/mespron...', Array) 
#10 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(431): Drupal\Core\Theme\ThemeManager->render('user-ranking', Array) 
#11 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(200): Drupal\Core\Render\Renderer->doRender(Array, false) 
#12 /var/www/htdocs/mybetsite/web/core/includes/common.inc(875): Drupal\Core\Render\Renderer->render(Array) 
#13 /var/www/htdocs/mybetsite/web/modules/mespronos/src/Controller/RankingController.php(130): render(Array) 
#14 /var/www/htdocs/mybetsite/web/modules/mespronos/src/Controller/RankingController.php(75): Drupal\mespronos\Controller\RankingController::getTableFromRanking(Array) 
#15 /var/www/htdocs/mybetsite/web/modules/mespronos/src/Controller/RankingController.php(58): Drupal\mespronos\Controller\RankingController::getRankingGeneral() 
#16 [internal function]: Drupal\mespronos\Controller\RankingController::rankingPage() 
#17 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(123): call_user_func_array(Array, Array) 
#18 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(573): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}() 
#19 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(124): Drupal\Core\Render\Renderer->executeInRenderContext(Object(Drupal\Core\Render\RenderContext), Object(Closure)) 
#20 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(97): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->wrapControllerExecutionInRenderContext(Array, Array) 
#21 /var/www/htdocs/mybetsite/vendor/symfony/http-kernel/HttpKernel.php(151): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}() 
#22 /var/www/htdocs/mybetsite/vendor/symfony/http-kernel/HttpKernel.php(68): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) 
#23 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/StackMiddleware/Session.php(57): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) 
#24 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/StackMiddleware/KernelPreHandle.php(47): Drupal\Core\StackMiddleware\Session->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) 
#25 /var/www/htdocs/mybetsite/web/core/modules/page_cache/src/StackMiddleware/PageCache.php(106): Drupal\Core\StackMiddleware\KernelPreHandle->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) 
#26 /var/www/htdocs/mybetsite/web/core/modules/page_cache/src/StackMiddleware/PageCache.php(85): Drupal\page_cache\StackMiddleware\PageCache->pass(Object(Symfony\Component\HttpFoundation\Request), 1, true) 
#27 /var/www/htdocs/mybetsite/web/modules/contrib/services/src/StackMiddleware/FormatSetter.php(44): Drupal\page_cache\StackMiddleware\PageCache->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) 
#28 /var/www/htdocs/mybetsite/web/core/modules/ban/src/BanMiddleware.php(50): Drupal\services\StackMiddleware\FormatSetter->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) 
#29 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/StackMiddleware/ReverseProxyMiddleware.php(47): Drupal\ban\BanMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) 
#30 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/StackMiddleware/NegotiationMiddleware.php(52): Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) 
#31 /var/www/htdocs/mybetsite/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Drupal\Core\StackMiddleware\NegotiationMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) 
#32 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/DrupalKernel.php(708): Stack\StackedHttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) 
#33 /var/www/htdocs/mybetsite/web/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request)) 
#34 {main} ``

It's strange because ranking seems to work just fine on mespronos.net site.
Maybe a server config problem, like php version or something?

**Comment by kgaut** [comment]
hi,

thanks for your repport, i'll try to remove external dependancies. And dig into your error message.

**Comment by bazio101** [comment]
Thanks for your answer.
I uninstall kgaut_tools and enabled error report in settings.php file.
This is the additional info on error page:

``Drupal::service('kgaut_tools.stringcleaner') (Line: 121) Drupal\mespronos\Controller\RankingController::getTableFromRanking(Array) (Line: 89) Drupal\mespronos\Controller\RankingController::getRankingLeague(Object) (Line: 29) Drupal\mespronos\Controller\LeagueController::getResultsAndRanking(Object) (Line: 318) mespronos_league_view(Array, Object, Object, 'full') call_user_func_array('mespronos_league_view', Array) (Line: 403) Drupal\Core\Extension\ModuleHandler->invokeAll('league_view', Array) (Line: 299) Drupal\Core\Entity\EntityViewBuilder->buildMultiple(Array) (Line: 250) Drupal\Core\Entity\EntityViewBuilder->build(Array) call_user_func_array(Array, Array) (Line: 100) Drupal\Core\Render\Renderer->doTrustedCallback(Array, Array, 'Render #pre_render callbacks must be methods of a class that implements \Drupal\Core\Security\TrustedCallbackInterface or be an anonymous function. The callback was %s. Support for this callback implementation is deprecated in 8.8.0 and will be removed in Drupal 9.0.0. See https://www.drupal.org/node/2966725', 'silenced_deprecation', 'Drupal\Core\Render\Element\RenderCallbackInterface') (Line: 781) Drupal\Core\Render\Renderer->doCallback('#pre_render', Array, Array) (Line: 372) Drupal\Core\Render\Renderer->doRender(Array, ) (Line: 200) Drupal\Core\Render\Renderer->render(Array, ) (Line: 226) Drupal\Core\Render\MainContent\HtmlRenderer->Drupal\Core\Render\MainContent\{closure}() (Line: 573) Drupal\Core\Render\Renderer->executeInRenderContext(Object, Object) (Line: 227) Drupal\Core\Render\MainContent\HtmlRenderer->prepare(Array, Object, Object) (Line: 117) Drupal\Core\Render\MainContent\HtmlRenderer->renderResponse(Array, Object, Object) (Line: 90) Drupal\Core\EventSubscriber\MainContentViewSubscriber->onViewRenderArray(Object, 'kernel.view', Object) call_user_func(Array, Object, 'kernel.view', Object) (Line: 111) Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher->dispatch('kernel.view', Object) (Line: 156) Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object, 1) (Line: 68) Symfony\Component\HttpKernel\HttpKernel->handle(Object, 1, 1) (Line: 57) Drupal\Core\StackMiddleware\Session->handle(Object, 1, 1) (Line: 47) Drupal\Core\StackMiddleware\KernelPreHandle->handle(Object, 1, 1) (Line: 191) Drupal\page_cache\StackMiddleware\PageCache->fetch(Object, 1, 1) (Line: 128) Drupal\page_cache\StackMiddleware\PageCache->lookup(Object, 1, 1) (Line: 82) Drupal\page_cache\StackMiddleware\PageCache->handle(Object, 1, 1) (Line: 44) Drupal\services\StackMiddleware\FormatSetter->handle(Object, 1, 1) (Line: 50) Drupal\ban\BanMiddleware->handle(Object, 1, 1) (Line: 47) Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle(Object, 1, 1) (Line: 52) Drupal\Core\StackMiddleware\NegotiationMiddleware->handle(Object, 1, 1) (Line: 23) Stack\StackedHttpKernel->handle(Object, 1, 1) (Line: 708) Drupal\Core\DrupalKernel->handle(Object) (Line: 19)``


Hope it helps.

**Comment by kgaut** [comment]
Hi ! 

The module should now works without my toolbox module. let me know

Thanks for the report.

This module date from the early alphas of drupal 8, I'll try to work to clean it a bit !

**Comment by bazio101** [comment]
I rebuild the site and this time I installed version 8.x-2.x-dev from drupal/modules site.
The errors I get now are the following (in the same pages, plus admin -view only- page):

**..../user/1**

Drupal\Core\Database\InvalidQueryException: Query condition 'file_managed.uuid IN ()' cannot be empty. στην Drupal\Core\Database\Query\Condition->condition() (line 105 of /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Database/Query/Condition.php)

**..../mespronos/leagues/1 &
..../mespronos/day/1**

Error: Call to a member function transformDimensions() on null in template_preprocess_image_style() (line 305 of /var/www/htdocs/mybetsite/web/core/modules/image/image.module)#0 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Theme/ThemeManager.php(287): template_preprocess_image_style(Array, 'image_style', Array) #1 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(431): Drupal\Core\Theme\ThemeManager->render('image_style', Array) #2 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(200): Drupal\Core\Render\Renderer->doRender(Array, false) #3 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Template/TwigExtension.php(501): Drupal\Core\Render\Renderer->render(Array) #4 /var/www/htdocs/mybetsite/web/sites/default/files/php/twig/5efde643eab25_user-ranking.html.twig_1-jlrZfk8YOhdOH8DxyK2lVIt/E5OLb3LBnsWB6kzsUQXlUzeJ_yP0sL2obJ0k3Q2Zm-4.php(63): Drupal\Core\Template\TwigExtension->escapeFilter(Object(Drupal\Core\Template\TwigEnvironment), Array, 'html', NULL, true) #5 /var/www/htdocs/mybetsite/vendor/twig/twig/src/Template.php(455): __TwigTemplate_6f453e5e1a67a2b9103ec444658514abe95599bfc88a47ec702803fc4fe57bc1->doDisplay(Array, Array) #6 /var/www/htdocs/mybetsite/vendor/twig/twig/src/Template.php(422): Twig\Template->displayWithErrorHandling(Array, Array) #7 /var/www/htdocs/mybetsite/vendor/twig/twig/src/Template.php(434): Twig\Template->display(Array) #8 /var/www/htdocs/mybetsite/web/core/themes/engines/twig/twig.engine(64): Twig\Template->render(Array) #9 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Theme/ThemeManager.php(384): twig_render_template('modules/mespron...', Array) #10 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(431): Drupal\Core\Theme\ThemeManager->render('user-ranking', Array) #11 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(200): Drupal\Core\Render\Renderer->doRender(Array, false) #12 /var/www/htdocs/mybetsite/web/core/includes/common.inc(875): Drupal\Core\Render\Renderer->render(Array) #13 /var/www/htdocs/mybetsite/web/modules/mespronos/src/Controller/RankingController.php(143): render(Array) #14 /var/www/htdocs/mybetsite/web/modules/mespronos/src/Controller/RankingController.php(103): Drupal\mespronos\Controller\RankingController::getTableFromRanking(Array) #15 /var/www/htdocs/mybetsite/web/modules/mespronos/src/Controller/LeagueController.php(40): Drupal\mespronos\Controller\RankingController::getRankingLeague(Object(Drupal\mespronos\Entity\League)) #16 /var/www/htdocs/mybetsite/web/modules/mespronos/src/Entity/ViewBuilder/LeagueViewBuilder.php(20): Drupal\mespronos\Controller\LeagueController::getResultsAndRanking(Object(Drupal\mespronos\Entity\League)) #17 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Entity/EntityViewBuilder.php(293): Drupal\mespronos\Entity\ViewBuilder\LeagueViewBuilder->buildComponents(Array, Array, Array, 'full') #18 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Entity/EntityViewBuilder.php(250): Drupal\Core\Entity\EntityViewBuilder->buildMultiple(Array) #19 [internal function]: Drupal\Core\Entity\EntityViewBuilder->build(Array) #20 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Security/DoTrustedCallbackTrait.php(100): call_user_func_array(Array, Array) #21 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(781): Drupal\Core\Render\Renderer->doTrustedCallback(Array, Array, 'Render #pre_ren...', 'silenced_deprec...', 'Drupal\\Core\\Ren...') #22 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(372): Drupal\Core\Render\Renderer->doCallback('#pre_render', Array, Array) #23 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(200): Drupal\Core\Render\Renderer->doRender(Array, false) #24 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/MainContent/HtmlRenderer.php(226): Drupal\Core\Render\Renderer->render(Array, false) #25 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(573): Drupal\Core\Render\MainContent\HtmlRenderer->Drupal\Core\Render\MainContent\{closure}() #26 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/MainContent/HtmlRenderer.php(227): Drupal\Core\Render\Renderer->executeInRenderContext(Object(Drupal\Core\Render\RenderContext), Object(Closure)) #27 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/MainContent/HtmlRenderer.php(117): Drupal\Core\Render\MainContent\HtmlRenderer->prepare(Array, Object(Symfony\Component\HttpFoundation\Request), Object(Drupal\Core\Routing\CurrentRouteMatch)) #28 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/EventSubscriber/MainContentViewSubscriber.php(90): Drupal\Core\Render\MainContent\HtmlRenderer->renderResponse(Array, Object(Symfony\Component\HttpFoundation\Request), Object(Drupal\Core\Routing\CurrentRouteMatch)) #29 [internal function]: Drupal\Core\EventSubscriber\MainContentViewSubscriber->onViewRenderArray(Object(Symfony\Component\HttpKernel\Event\GetResponseForControllerResultEvent), 'kernel.view', Object(Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher)) #30 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php(111): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\GetResponseForControllerResultEvent), 'kernel.view', Object(Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher)) #31 /var/www/htdocs/mybetsite/vendor/symfony/http-kernel/HttpKernel.php(156): Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher->dispatch('kernel.view', Object(Symfony\Component\HttpKernel\Event\GetResponseForControllerResultEvent)) #32 /var/www/htdocs/mybetsite/vendor/symfony/http-kernel/HttpKernel.php(68): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #33 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/StackMiddleware/Session.php(57): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #34 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/StackMiddleware/KernelPreHandle.php(47): Drupal\Core\StackMiddleware\Session->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #35 /var/www/htdocs/mybetsite/web/core/modules/page_cache/src/StackMiddleware/PageCache.php(106): Drupal\Core\StackMiddleware\KernelPreHandle->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #36 /var/www/htdocs/mybetsite/web/core/modules/page_cache/src/StackMiddleware/PageCache.php(85): Drupal\page_cache\StackMiddleware\PageCache->pass(Object(Symfony\Component\HttpFoundation\Request), 1, true) #37 /var/www/htdocs/mybetsite/web/modules/contrib/services/src/StackMiddleware/FormatSetter.php(44): Drupal\page_cache\StackMiddleware\PageCache->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #38 /var/www/htdocs/mybetsite/web/core/modules/ban/src/BanMiddleware.php(50): Drupal\services\StackMiddleware\FormatSetter->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #39 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/StackMiddleware/ReverseProxyMiddleware.php(47): Drupal\ban\BanMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #40 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/StackMiddleware/NegotiationMiddleware.php(52): Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #41 /var/www/htdocs/mybetsite/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Drupal\Core\StackMiddleware\NegotiationMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #42 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/DrupalKernel.php(708): Stack\StackedHttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #43 /var/www/htdocs/mybetsite/web/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request)) #44 {main}


**..../mespronos/ranking**

Error: Call to a member function transformDimensions() on null in template_preprocess_image_style() (line 305 of /var/www/htdocs/mybetsite/web/core/modules/image/image.module)#0 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Theme/ThemeManager.php(287): template_preprocess_image_style(Array, 'image_style', Array) #1 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(431): Drupal\Core\Theme\ThemeManager->render('image_style', Array) #2 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(200): Drupal\Core\Render\Renderer->doRender(Array, false) #3 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Template/TwigExtension.php(501): Drupal\Core\Render\Renderer->render(Array) #4 /var/www/htdocs/mybetsite/web/sites/default/files/php/twig/5efde643eab25_user-ranking.html.twig_1-jlrZfk8YOhdOH8DxyK2lVIt/E5OLb3LBnsWB6kzsUQXlUzeJ_yP0sL2obJ0k3Q2Zm-4.php(63): Drupal\Core\Template\TwigExtension->escapeFilter(Object(Drupal\Core\Template\TwigEnvironment), Array, 'html', NULL, true) #5 /var/www/htdocs/mybetsite/vendor/twig/twig/src/Template.php(455): __TwigTemplate_6f453e5e1a67a2b9103ec444658514abe95599bfc88a47ec702803fc4fe57bc1->doDisplay(Array, Array) #6 /var/www/htdocs/mybetsite/vendor/twig/twig/src/Template.php(422): Twig\Template->displayWithErrorHandling(Array, Array) #7 /var/www/htdocs/mybetsite/vendor/twig/twig/src/Template.php(434): Twig\Template->display(Array) #8 /var/www/htdocs/mybetsite/web/core/themes/engines/twig/twig.engine(64): Twig\Template->render(Array) #9 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Theme/ThemeManager.php(384): twig_render_template('modules/mespron...', Array) #10 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(431): Drupal\Core\Theme\ThemeManager->render('user-ranking', Array) #11 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(200): Drupal\Core\Render\Renderer->doRender(Array, false) #12 /var/www/htdocs/mybetsite/web/core/includes/common.inc(875): Drupal\Core\Render\Renderer->render(Array) #13 /var/www/htdocs/mybetsite/web/modules/mespronos/src/Controller/RankingController.php(143): render(Array) #14 /var/www/htdocs/mybetsite/web/modules/mespronos/src/Controller/RankingController.php(89): Drupal\mespronos\Controller\RankingController::getTableFromRanking(Array) #15 /var/www/htdocs/mybetsite/web/modules/mespronos/src/Controller/RankingController.php(70): Drupal\mespronos\Controller\RankingController::getRankingGeneral() #16 [internal function]: Drupal\mespronos\Controller\RankingController::rankingPage() #17 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(123): call_user_func_array(Array, Array) #18 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/Render/Renderer.php(573): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}() #19 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(124): Drupal\Core\Render\Renderer->executeInRenderContext(Object(Drupal\Core\Render\RenderContext), Object(Closure)) #20 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(97): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->wrapControllerExecutionInRenderContext(Array, Array) #21 /var/www/htdocs/mybetsite/vendor/symfony/http-kernel/HttpKernel.php(151): Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}() #22 /var/www/htdocs/mybetsite/vendor/symfony/http-kernel/HttpKernel.php(68): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #23 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/StackMiddleware/Session.php(57): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #24 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/StackMiddleware/KernelPreHandle.php(47): Drupal\Core\StackMiddleware\Session->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #25 /var/www/htdocs/mybetsite/web/core/modules/page_cache/src/StackMiddleware/PageCache.php(106): Drupal\Core\StackMiddleware\KernelPreHandle->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #26 /var/www/htdocs/mybetsite/web/core/modules/page_cache/src/StackMiddleware/PageCache.php(85): Drupal\page_cache\StackMiddleware\PageCache->pass(Object(Symfony\Component\HttpFoundation\Request), 1, true) #27 /var/www/htdocs/mybetsite/web/modules/contrib/services/src/StackMiddleware/FormatSetter.php(44): Drupal\page_cache\StackMiddleware\PageCache->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #28 /var/www/htdocs/mybetsite/web/core/modules/ban/src/BanMiddleware.php(50): Drupal\services\StackMiddleware\FormatSetter->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #29 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/StackMiddleware/ReverseProxyMiddleware.php(47): Drupal\ban\BanMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #30 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/StackMiddleware/NegotiationMiddleware.php(52): Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #31 /var/www/htdocs/mybetsite/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Drupal\Core\StackMiddleware\NegotiationMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #32 /var/www/htdocs/mybetsite/web/core/lib/Drupal/Core/DrupalKernel.php(708): Stack\StackedHttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #33 /var/www/htdocs/mybetsite/web/index.php(19): Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request)) #34 {main}

**Web Server:**
Apache/2.4.43
PHP/7.1.28
MariaDB/10.0.38

Is it possible to be a php version problem? 
Does mespronos module need 7.3.x or higher?
I suspect, I should upgrade some things soon...
Thanks again for your patience.

**Comment by bazio101** [comment]
Last time I installed mespronos 2.x-dev from admin environment.
Today I created a new (version 8.9.1) site using composer and when I try to install module with command:
composer require 'drupal/mespronos:2.x-dev',
I get:
[InvalidArgumentException]
Package drupal/mespronos at version 2.x-dev has a PHP requirement incompatible with your PHP version (7.1.28)
So, I'll upgrade my PHP version and try again.
I close this issue.
Thanks again.
